proto/iot.proto

```diff
@@ -119,6 +126,34 @@ message Thermostat {
     Double cooling_setpoint = 11;
 }
 
+//Schedule of one week
+message Schedule {
+    //Map day of week to schedule
+    map<string, ScheduleDay> scheduleMap = 1;
+}
+
+// Repeated Schedule Wrapper
+message ScheduleDay {
+    //Series of Schedule Blocks
+    repeated ScheduleBlock blocks = 1;
+}
+
+// Schedule Block
+message ScheduleBlock {
+    //heating setpoint
+    //unit: celsius
+    Double heat_setting = 1;
+    //cooling setpoint
+    //unit: celsius
+    Double cool_setting = 2;
+    //Current system mode of thermostat
+    //unit: xbos/iot/SystemMode
+    SystemMode mode = 3;
```

Comment:
Switch to the `HVACMode` enum as indicated above

Reply:
👍 also made sure to delete the SystemMode struct, too.